addon-resizer/nanny/nanny_lib.go

```diff
@@ -168,23 +168,21 @@ func updateResources(k8s KubernetesClient, est ResourceEstimator, now, lastChang
 	// If there's a difference, go ahead and set the new values.
 	overwriteRes, op := shouldOverwriteResources(int64(threshold), resources.Limits, resources.Requests, expResources.Limits, expResources.Requests)
 	if !overwriteRes {
-		if log.V(4) {
```

Comment:
This change could have performance implications. Does klog have the if option too?

Reply:
Replaced with klog and it has error `non-boolean condition in if statement` when trying to build the image

Reply:
ohh looks like I should use `if klog.V(2).Enabled() `

Reply:
Done.